<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeleoperationWidget</class>
 <widget class="QWidget" name="TeleoperationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>146</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="forwardButton">
     <property name="text">
      <string>↑</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="leftButton">
     <property name="text">
      <string>←</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="backButton">
     <property name="text">
      <string>↓</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="rightButton">
     <property name="text">
      <string>→</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Left wheel power</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Right wheel power</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLCDNumber" name="leftSpeedLCD"/>
   </item>
   <item row="3" column="2">
    <widget class="QLCDNumber" name="rightSpeedLCD"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="toAerialButton">
     <property name="text">
      <string>Switch to aerial</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="kenafSpeedSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
